### ISSUE TITLE
Messages showing backwards

### ISSUE THREAD

**Issue by GreenBayRules** [LLK/scratch-www#1631]
For example:
![image](https://user-images.githubusercontent.com/11379161/32029489-c72600a2-b9a9-11e7-9d1c-747e385333ce.png)
For some reason the "today" is appearing as "yadot" I have seen this before multiple times, but I have never reported it.

**Comment by towerofnix** [comment]
Could you post a link to the studio here?

It's possible that the studio has an RTL override inside its name. [My studio](https://scratch.mit.edu/studios/3004831/activity), which has that character in its name, also shows up with "yadot":

![Screenshot of lol rtl message row](https://user-images.githubusercontent.com/9948030/32030621-83b549b2-b9d2-11e7-95c7-a456ad3b1b27.png)

It would be weird if *any* fancy characters caused it to flip around, though.

**Comment by Kenny2github** [comment]
Yeah, it looks like if the studio has a RTL character of any kind (including the RTL override) the "today" in "new activity in (studio) today" is flipped around, i.e. "new activity in lol rtl today" becomes "new activity in yadot lol rtl".

**Comment by GreenBayRules** [comment]
@towerofnix Here is the studio name: https://scratch.mit.edu/studios/4043166/

**Comment by joker314** [comment]
Based on the studio activity, the title of the studio may have changed since this issue was filed.

**Comment by GreenBayRules** [comment]
@joker314 The title of the studio that was causing this was "☄ⓁⒺ ⒼⓇⓄⓊⓅ☄"

**Comment by mewtaylor** [comment]
Thanks y'all for investigating!

@chrisgarrity it seems like we might need to start adding overrides to our strings that have variable content in them to ensure that the ltr text stays ltr regardless of the variable content.

**Comment by chrisgarrity** [comment]
@mewtaylor mixing rtl and ltr is always a challenge - definitely something to look into.
